TODO/designing-the-icons-for-flinto-s-ui.md

```diff
@@ -1,86 +1,83 @@
 
 ![](http://ww3.sinaimg.cn/large/a490147fjw1f2m0k0slikj20m807xjsz.jpg)
 
-<figcaption>**“Half Size”** bitmaps shown for readers using a retina display, to see the 1x icon in full pixels. **“Double Size”** shown for readers using a non-retina display, to see the 2x icon in full pixels. Best viewed on a landscape tablet or larger screen.</figcaption>
+<figcaption>对于使用retina显示屏的读者，我们显示**“一半尺寸”**的位图，如图1x的全像素图标。对于非retina显示屏的读者，则使用**“双倍尺寸”**的位图，如图2x的全像素图标。 请以横向模式显示上图来获得最好的显示效果。</figcaption>
 
-In an ideal world, a well-made vector icon can simply be exported at various pixel-densities and look great at all of those scales. But much of the time, the limitations of designing a 1x asset aren’t worth carrying over to higher pixel densities. You may be able to build an icon perfectly at 2x, but need to make adjustments to create a decent and crisp 1x asset. At least half of the icons in Flinto’s UI have unique 1x and 2x versions, such as the “Connect Layers” icon throughout the Transition Designer.
+在一个理想的世界中，一个做工精良的向量图标能够简单的导出其任何对应尺寸的图标。但是大部分时候，使用一倍大小的图标并不能够处理得到更高尺寸的图标。你可能需要先做一个完美的两倍尺寸的图标，然后再调整成一倍尺寸来创建一个新的视觉满意的图标。在Flinto中至少一半的图标都有其对应1倍和2倍尺寸，譬如贯穿整个过渡动画设计面板的"概念图层"图标。
 
 ![](http://ww4.sinaimg.cn/large/a490147fjw1f2m0ksytpgj20m808wq5a.jpg)
 
-**For anyone who is curious, here are some techniques I used for fine tuning the antialiasing of icons in Flinto:**
+**针对于这块感兴趣的读者，这里有我是如何对Flinto图标的抗锯齿进行细调的技术细节。**
 
-*   Resize and reposition shapes for crisper results, even if that means the position or dimensions have decimals. It’s about appearance, not values.
-*   Only use curves or rounded corners when there are at least 2px to render each 90° angle of that radius’s circle (see below), or 3px for 180°, as with a rounded ends of a line (below). Goodbye, rounded 1pt line caps — at least until we’re all using 3x screens.
+*  重新调整和重新定位图形来获得看上去视觉舒服的图形，尽管这样处理后位置或者像素值会有小数点，但在这个阶段视觉是重点。
+*  只使用曲线或者圆角时，至少要使用2px来渲染 90° 角的半径圆，或者使用3px渲染180°角的半径圆，来作为园的线段末尾（如下图）。1pt大小线的线段的圆角线帽的效果是很糟糕的，至少我们使用的屏幕都会将其放大三倍来显示。
 
 ![](http://ww4.sinaimg.cn/large/a490147fjw1f2m0lacz7xj20m80bmdha.jpg)
 
-<figcaption>Nobody wants fuzzy line caps! Only a 3x screen (or higher) can render a decipherable rounded line cap on a 1pt line.</figcaption>
+<figcaption>没有人想要模糊的线帽！只有三倍大小（或者更大）的显示器才能够对1pt大小的线条渲染清晰可见的圆角线帽。</figcaption>
 
-*   Adjust border width/thickness to be slightly wider or thinner than 1pt on thin curved or diagonal lines, for a more consistent perceived thickness.
-*   Mask out unwanted blurry pixels. This will even have an effect if a shape is masked out by a duplicate of itself (no change in shape).
-*   Duplicate a shape or its border (same position) to be slightly bolder.
-*   Determine when it’s ok to have a little antialiasing fuzziness because a more important part of the icon benefits. It’s going to be ok!
+*  为了让线条粗细更加一致，调整边框宽度/厚度来达到稍宽或者稍厚会比使用1pt的细曲线或者斜线更好。
+*  标记上不需要的模糊像素。这在你需要使用图形自身标记自己的时候将会很有效。
+*  通过复制图形或者边框（同一方向）来轻微调整图形的粗细。
+*  如果图标可以有小模糊锯齿能够为图标的其他部分提供一定帮助，这也是可以的。
```

Comment:
_Determine when it’s ok to have a little antialiasing fuzziness because a more important part of the icon benefits. _
建议翻译：确定在对图标效果大有帮助的前提下可以允许轻微抗锯齿模糊的存在。